packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -321,8 +321,12 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
 }
 
 class _AppBarState extends State<AppBar> {
-  void _handleDrawerButton() {
-    Scaffold.of(context).openDrawer();
+  void _handleDrawerButtonLeft() {
+    Scaffold.of(context).openLeftDrawer();
```

Comment:
These two functions still refer to the "left" drawer, which isn't an accurate name in languages with right-to-left reading orders.

Reply:
Fixed